src/entity/sources/SourceReport.ts

```diff
@@ -0,0 +1,29 @@
+import { Column, Entity, Index, ManyToOne, PrimaryColumn } from 'typeorm';
+import type { User } from '../user';
+import { ReportReason } from '../common';
+
+@Entity()
+export class SourceReport {
+  @PrimaryColumn({ type: 'text' })
+  @Index('IDX_source_report_source_id')
+  sourceId: string;
+
+  @PrimaryColumn({ length: 36 })
+  @Index('IDX_source_report_user_id')
+  userId: string;
+
+  @Column({ default: () => 'now()' })
+  createdAt: Date;
+
+  @Column({ length: 32, type: 'varchar' })
```

Comment:
I think we make it text lately since varchar and text are basically the same in psql.

Reply:
same for other columns

Reply:
Sure, let's update these ones to that.

Reply:
Updated this one, but for the others, I had to keep it, the migration says it needs to drop the column first to change the type.